portal-web/test/functional/com/liferay/portalweb/tests/enduser/headless/AdvancedObjectAPI/AdvancedObjectAPI.testcase

```diff
@@ -0,0 +1,71 @@
+@component-name = "portal-headless"
+definition {
+
+	property portal.release = "true";
+	property portal.upstream = "true";
+	property testray.main.component.name = "Headless";
+
+	setUp {
+		TestCase.setUpPortalInstance();
+
+		task ("Given active object definitions created") {
+			var objectDefinitionId1 = ObjectDefinitionAPI.createAndPublishObjectDefinition(
+				en_US_label = "Manager",
+				en_US_plural_label = "Managers",
+				name = "Manager",
+				requiredStringFieldName = "firstname");
+
+			var objectDefinitionId2 = ObjectDefinitionAPI.createAndPublishObjectDefinition(
+				en_US_label = "Employee",
+				en_US_plural_label = "Employees",
+				name = "Employee",
+				requiredStringFieldName = "firstname");
+		}
+
+		task ("Given a relationship between two object definitions created") {
+			var relationshipName = ObjectDefinitionAPI.createRelationship(
+				deletionType = "cascade",
+				en_US_label = "Supervisor",
+				name = "supervisor",
+				objectDefinitionId1 = "${objectDefinitionId1}",
+				objectDefinitionId2 = "${objectDefinitionId2}",
+				type = "oneToMany");
+		}
+	}
+
+	tearDown {
+		PortalInstances.tearDownCP();
+	}
+
+	@description = "Nested fields in oneToMany relationship"
+	@priority = "5"
+	test NestedFieldsInOneToManyRelationship {
```

Comment:
I wonder how we can make this test name more SACRED to say what feature/behavior/capability we're testing.